scripts/compare.py

```diff
@@ -0,0 +1,199 @@
+# /// script
+# requires-python = ">=3.12"
+# dependencies = [
+#     "inflection",
+#     "requests",
+#     "rich",
+# ]
+# ///
```

Comment:
pretty nifty - this was generated by:

```
➜ uv add --script ./scripts/compare.py requests inflection rich
```

that way, we dont need a requirements.txt and can just run

```
➜ uv run ./scripts/compare.py
```